pandas/io/formats/excel.py

```diff
@@ -402,6 +402,10 @@ def _format_value(self, val):
                 val = '-{inf}'.format(inf=self.inf_rep)
             elif self.float_format is not None:
                 val = float(self.float_format % val)
+        if getattr(val, 'tzinfo', None) is not None:
+            raise ValueError('Excel does not support datetimes with timzones. '
```

Comment:
small typo here for timezones if you want to fix locally (otherwise can do before merge)